conda_build/__init__.py

```diff
@@ -15,3 +16,17 @@
     "render",
     "skeleton",
 ]
+
+# Skip context logic for doc generation since we don't install all dependencies in the CI doc build environment,
+# see .readthedocs.yml file
```

Comment:
I would argue this is a bug and needs to be resolved by installing all dependencies in the Ci doc build environment. Maybe add caching there? Switch to setup-miniconda?

Reply:
Sure but that will require that we stop using the RTD pipeline and switch to a custom workflow which is out of scope for this PR